db/view/view.cc

```diff
@@ -342,7 +342,7 @@ class view_updates final {
             , _updates(8, partition_key::hashing(*_view), partition_key::equality(*_view)) {
     }
 
-    void move_to(std::vector<frozen_mutation_and_schema>& mutations) && {
+    void move_to(utils::chunked_vector<frozen_mutation_and_schema>& mutations) && {
         std::transform(_updates.begin(), _updates.end(), std::back_inserter(mutations), [&, this] (auto&& m) {
             auto mut = mutation(_view, dht::decorate_key(*_view, std::move(m.first)), std::move(m.second));
```

Comment:
If the vector can be big, can't this stall?

Reply:
Yeah, it's another candidate for futurizing I guess. Or perhaps this whole machinery should be able to return to results in chunks instead of all at once.

Reply:
We need to generate a stream of updates and then process them. I need to revive my idea of supporting

```c++
stream<blah> make_blahs() {
   while (...) {
       co_yield make_a_blah(); 
   }
}
```

and

```c++
future<> eat_blahs(stream<blah> s) {
    while (std::optional<blah> b = co_await s) {
         // do something with *b
    }
}
```